src/api/v1beta1/properties.go

```diff
@@ -38,7 +38,7 @@ const (
 	AuthTokenSecretSuffix                   = "-activegate-authtoken-secret"
 	PodNameOsAgent                          = "oneagent"
 
-	defaultActiveGateImage = "/linux/activegate:latest"
+	defaultActiveGateImage = "/linux/activegate:raw"
```

Comment:
should the default image include the tag or can we add that when building the final image?